doc/manuals-ld/installation-guide-debian9.md

```diff
@@ -221,7 +221,14 @@ make html
 make
 sudo make install
 
+
 # Python library
+
+# [ If you don't have pip installed: ]
+sudo aptitude update
+sudo aptitude install python-pip
+pip --version
```

Comment:
If we ask the user to issue a "pip --version" command, we should also tell them what should be the desired output of the command.

Either that, or just remove the version line.

Reply:
Okay